src/engraving/libmscore/noteentry.cpp

```diff
@@ -387,8 +387,8 @@ void Score::putNote(const Position& p, bool replace)
     staff_idx_t staffIdx = track2staff(_is.track());
     if (m->isMeasureRepeatGroup(staffIdx)) {
         auto b = MessageBox::warning(trc("engraving", "Note input will remove measure repeat"),
-                                     trc("engraving", "There is a measure repeat here.")
-                                     + trc("engraving", "\nContinue with adding note and delete measure repeat?"));
+                                     trc("engraving", "There is a measure repeat here.") + "/n"
+                                     + trc("engraving", "Continue with adding note and delete measure repeat?"));
```

Comment:
Actually, do we want a line break here at all? It looks like one continuous text. Personally I would change it like this:
```suggestion
                                     trc("engraving", "There is a measure repeat here. If you enter notes here, "
                                                      "it will be deleted. Do you want to continue?"));
```
(The string itself shouldn't contain a line break, I'm just spreading the literal over two lines.)

Reply:
Yes, I'd agree here